src/D2L.Bmx/BmxPaths.cs

```diff
@@ -1,11 +1,14 @@
 namespace D2L.Bmx;
 
 internal static class BmxPaths {
-	public static readonly string USER_HOME_DIR = Environment.GetFolderPath( Environment.SpecialFolder.UserProfile );
-	public static readonly string BMX_DIR = Path.Join( USER_HOME_DIR, ".bmx" );
-	public static readonly string CACHE_DIR = Path.Join( BMX_DIR, "\\cache" );
-	public static readonly string SESSIONS_FILE_NAME = Path.Join( CACHE_DIR, "sessions" );
+	private static readonly string BMX_DIR = Path.Join(
+		Environment.GetFolderPath( Environment.SpecialFolder.UserProfile ),
```

Comment:
`USER_HOME_DIR` and `BMX_DIR` are only used inside this static class to define other paths now, so they should be private or inlined.